packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -80,6 +80,15 @@ export interface ConfigurationSetProps {
    * @default - VDM options not configured at the configuration set level. In this case, use account level settings. (To set the account level settings using CDK, use the `VdmAttributes` Construct.)
    */
   readonly vdmOptions?: VdmOptions;
+
+  /**
+   * The maximum amount of time that Amazon SES API v2 will attempt delivery of email.
+   *
+   * This value must be greater than or equal to 5 minutes and less than or equal to 14 hours.
+   *
+   * @default - 14 hours
```

Comment:
This default value is described at [Advanced delivery options/Maximum delivery duration section](https://docs.aws.amazon.com/ses/latest/dg/creating-configuration-sets.html#config-sets-create-console).

```Setting a custom maximum delivery limit (shorter than the SES default of 14 hours),```